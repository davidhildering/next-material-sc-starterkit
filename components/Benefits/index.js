import React from "react";
import { Container, Row, Col } from "react-grid-system";
import styled from "styled-components";
import Button from "../Button";
import { Wrapper } from "../Wrapper";
import { Header3, Paragraph } from "../Typography";

const Benefit = styled.div`
  width: 100%;
  background: #ffffff;
  padding: 16px;
  margin-bottom: 16px;
  text-align: center;
  height: 300px;
  border-radius: 16px;
  box-sizing: border-box;
  img {
    padding: 16px 0;
    width: 80px;
  }
`;

const ButtonContainer = styled.div`
  width: 100%;
  max-width: 200px;
  margin: 40px auto;
`;

const Benefits = () => (
  <Wrapper id="benefits" color="#ecf5f5">
    <Container>
      <Header3>De Voordelen</Header3>
      <Row>
        <Col md={4}>
          <Benefit>
            <Paragraph>
              "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              eiusmod tempor incididunt.
            </Paragraph>
          </Benefit>
        </Col>
        <Col md={4}>
          <Benefit>
            <Paragraph>
              "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              eiusmod tempor incididunt.
            </Paragraph>
          </Benefit>
        </Col>
        <Col md={4}>
          <Benefit>
            <Paragraph>
              "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              eiusmod tempor incididunt.
            </Paragraph>
          </Benefit>
        </Col>
      </Row>
    </Container>
    <ButtonContainer>
      <Button title="CTA make appointment" />
    </ButtonContainer>
  </Wrapper>
);
export default Benefits;
