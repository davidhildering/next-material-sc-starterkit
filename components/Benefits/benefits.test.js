import React from "react";
import { shallow } from "enzyme";
import Benefits from ".";

describe("Benefits", () => {
  it("should render correctly with no props", () => {
    const component = shallow(<Benefits />);

    expect(component).toMatchSnapshot();
  });
});
