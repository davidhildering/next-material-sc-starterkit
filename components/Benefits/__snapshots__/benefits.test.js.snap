// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Benefits should render correctly with no props 1`] = `
<Wrapper
  color="#ecf5f5"
  id="benefits"
>
  <Container
    component="div"
    fluid={false}
    lg={false}
    md={false}
    sm={false}
    style={Object {}}
    xl={false}
    xs={false}
  >
    <Typography__Header3>
      De Voordelen
    </Typography__Header3>
    <Row
      align="normal"
      component="div"
      debug={false}
      justify="start"
      nogutter={false}
      style={Object {}}
    >
      <Col
        component="div"
        debug={false}
        lg={null}
        md={4}
        offset={Object {}}
        pull={Object {}}
        push={Object {}}
        sm={null}
        style={Object {}}
        xl={null}
        xs={null}
      >
        <Benefits__Benefit>
          <Typography__Paragraph>
            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </Typography__Paragraph>
        </Benefits__Benefit>
      </Col>
      <Col
        component="div"
        debug={false}
        lg={null}
        md={4}
        offset={Object {}}
        pull={Object {}}
        push={Object {}}
        sm={null}
        style={Object {}}
        xl={null}
        xs={null}
      >
        <Benefits__Benefit>
          <Typography__Paragraph>
            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </Typography__Paragraph>
        </Benefits__Benefit>
      </Col>
      <Col
        component="div"
        debug={false}
        lg={null}
        md={4}
        offset={Object {}}
        pull={Object {}}
        push={Object {}}
        sm={null}
        style={Object {}}
        xl={null}
        xs={null}
      >
        <Benefits__Benefit>
          <Typography__Paragraph>
            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </Typography__Paragraph>
        </Benefits__Benefit>
      </Col>
    </Row>
  </Container>
  <Benefits__ButtonContainer>
    <Button
      title="Afspraak maken adviseur"
    />
  </Benefits__ButtonContainer>
</Wrapper>
`;
